src/bin.js

```diff
@@ -23,9 +23,9 @@ if (!isAcceptedNodeVersion()) {
   }
 
   mochaOpts.forEach(p => writeMochaOpts(p));
-})();
 
-// TOOD: allow passing a specific path
-writePackageJSON();
-// TODO: just do this automatically
-console.log("You'll need to re-run yarn or npm install");
+  // TOOD: allow passing a specific path
+  await writePackageJSON();
```

Comment:
Is this part correct? Presumably we only want to install deps once writePackageJSON has finished.

Reply:
yep for sure

Reply:
we should figure out how to test src/bin.js though